### PR TITLE
[COST-4745] Add `data_transfer_direction` to GCP network costs to Trino tables

### PR DESCRIPTION
**Pull request by samdoran**
## Jira Ticket

[COST-4745](https://issues.redhat.com/browse/COST-4745)

## Description

Add `data_transfer_direction` to Trino tables for tracking GCP network costs.

## Testing

1. Checkout Branch
2. Delete existing data
3. Load OCP on GCP data

## Release Notes
- [x] proposed release note

```markdown
* [COST-4745](https://issues.redhat.com/browse/COST-4745) Add `data_transfer_direction` for GCP network costs to Trino tables 
```

### CODE REVIEW

File: koku/masu/database/trino_sql/gcp/openshift/reporting_ocpgcpcostlineitem_daily_summary_resource_id.sql [hunk 1]
```diff
@@ -38,6 +39,7 @@ CREATE TABLE IF NOT EXISTS {{schema | sqlsafe}}.gcp_openshift_daily_tag_matched_
     instance_type varchar,
     service_id varchar,
     service_alias varchar,
+    data_transfer_direction varchar,
```
Comment (cgoodfred) [inline koku/masu/database/trino_sql/gcp/openshift/reporting_ocpgcpcostlineitem_daily_summary_resource_id.sql:42]:
```suggestion
```
This needs to be added within the creation of `reporting_ocpgcpcostlineitem_project_daily_summary` not `gcp_openshift_daily_tag_matched_temp`
Relevant reference: https://github.com/project-koku/koku/pull/5058/files#diff-d93349900fcf45dec5595ef5c01c5f394889315c807c935062130dcc6b538f8cR138

**Review by cgoodfred — APPROVED** [review]

**Review by cgoodfred — CHANGES_REQUESTED** [review]
This was still added to the wrong table, it should have been added to the non temp table. The same line in the link here: https://github.com/project-koku/koku/pull/5058/files#diff-d93349900fcf45dec5595ef5c01c5f394889315c807c935062130dcc6b538f8cR138

**Comment by samdoran** [comment]
I'll fix this up tomorrow.

**Comment by samdoran** [comment]
I added the field to the correct table.

**Review by cgoodfred — APPROVED** [review]

**Pull request merged**